lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts

```diff
@@ -195,7 +195,7 @@ export class DropdownCloudWidgetComponent extends WidgetComponent implements OnI
     }
 
     private setFormControlValue(): void {
-        this.dropdownControl.setValue(this.field?.value, { emitEvent: false });
+        this.dropdownControl.setValue({ id: this.field?.value, name: '' }, { emitEvent: false });
```

Comment:
Do not understand this, we should set value but we are updating a `id` and `name` 🤔 

If mapping is not working, maybe just calling `updateFormControlState` would be enough?  it will revalidate field again?

cc @tomgny  I think you have some more knowledge about it WDYT?

Reply:
that form control should have id and name. we set it in this way when it come from the mapping correct

Reply:
Ok got it, dropdown value should point to dropdown option using `option id`.

I assume that previous code `this.dropdownControl.setValue(this.field?.value..),` works fine i other cases, therefore `this.field?.value` should had correct object `{id, name}`.

I was thinking if we could map value from mapping into correct object { id, name  }?  instead of updating it here? So we wont break other case (e.g linking dropdowns, async dropdowns, manual value update etc hope we have e2e for it 🤞 )

Reply:
i can try

Reply:
same that doesn't make our plain an object